### PR TITLE
feat: addWebpackExternals

### PR DESCRIPTION
**Pull request by zhigang1992**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -150,6 +150,10 @@ Adds the provided alias info into webpack's alias section. Pass an object litera
 
 Adds the provided resolve info into webpack's resolve section. Pass an object literal with as many entries as you'd like, and the whole object will be merged in.
 
+### addWebpackExternals(deps)
+
+Add external dependencies, useful when trying to offload libs to CDN
```
Comment (FezVrasta) [inline readme.md]: May you provide a short example to explain how to use this?

Comment (zhigang1992) [reply readme.md]: Here you go 

https://webpack.js.org/configuration/externals/
https://github.com/facebook/create-react-app/issues/2758

Should I add them to the documents?

Comment (FezVrasta) [reply readme.md]: Ideally the example should be written directly in the document

Comment (zhigang1992) [reply readme.md]: Also firebase js sdk https://firebase.google.com/docs/web/setup#reserved-urls with externals we can import firebase in js file also using the js firebase provided

**Comment by zhigang1992** [comment]
@FezVrasta docs added. 😄

**Comment by FezVrasta** [comment]
Great thanks!

**Pull request merged**